nifi-frontend/src/main/frontend/apps/update-attribute/src/app/pages/update-attribute/ui/action-table/action-table.component.html

```diff
@@ -0,0 +1,150 @@
+<!--
+  ~ Licensed to the Apache Software Foundation (ASF) under one or more
+  ~ contributor license agreements.  See the NOTICE file distributed with
+  ~ this work for additional information regarding copyright ownership.
+  ~ The ASF licenses this file to You under the Apache License, Version 2.0
+  ~ (the "License"); you may not use this file except in compliance with
+  ~ the License.  You may obtain a copy of the License at
+  ~
+  ~     http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing, software
+  ~ distributed under the License is distributed on an "AS IS" BASIS,
+  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~ See the License for the specific language governing permissions and
+  ~ limitations under the License.
+  -->
+
+<div class="action-table flex flex-col h-full gap-y-2">
+    <div class="flex justify-end items-center">
+        @if (!isDisabled) {
+            <button mat-icon-button class="primary-icon-button" type="button" (click)="newActionClicked()">
```

Comment:
Try adding 7 or 8 of these. Then save it. Some get saved and some do not. Do we want to allow the user to save 'empty' attribute/value pairs?